<?xml version="1.0"?>
<launch>
    <node name="route_navigation" pkg="route_navigation" type="demo_navigation" output="screen">
        <param name="move_base_server" value="move_base" />
        <param name="move_base_feedback_topic" value="/move_base/feedback" />
        <param name="move_base_cancel_topic" value="/move_base/cancel" />
    </node>
</launch>
